splunkapp/generate_wazuh_splunk_app.sh

```diff
@@ -131,6 +133,10 @@ main() {
                     help 1 
                 fi
                 ;;
+            "-k" | "--checksum")
+                cheksum="yes"
```

Comment:
Typo => `cheksum`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/ba5843f8aac9201fe518b0d257c88d6493431ec2 and https://github.com/wazuh/wazuh-packages/pull/192/commits/5de766ec4dd5cddd21718c5897f9431117361be8